### PR TITLE
Added 2023 bookings link

### PR DESCRIPTION
**Pull request by jdfergusson**
DO NOT MERGE until bookings are actually open. But please review!

### CODE REVIEW

**Comment by jdfergusson** [comment]
@tjvr Some weird formatting is happening on a paragraph on the tech page towards the bottom:
image
Any ideas?

**Review by jamesdoc — APPROVED** [review]

**Comment by jamesdoc** [comment]
That paragraph looks fine from this end…

Screenshot 2023-01-11 at 11 22 16

**Comment by jamesdoc** [comment]
At a guess, perhaps the browser you're using has an bug with the CSS columns? @tjvr, might be worth swaping out with `grid` which is generally nicer and the future ™️. Grid doesn't have masonry yet, but it is coming with `grid-template-rows: masonry`.

Screenshot 2023-01-11 at 11 25 18
Screenshot 2023-01-11 at 11 25 06

**Comment by tjvr** [comment]
> might be worth swaping out with grid which is generally nicer and the future

I defer to your expertise here! :)

**Comment by tjvr** [comment]
Changes look good to me too.

File: pages/index.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ og_description: "Start your summer with a week learning about Jesus and technolo
 
 * Technical activity week for 12–15s
 * 6th - 13th August 2023
-* Chafyn Grove School, Salisbury, UK
```
Comment (jamesdoc) [inline pages/index.md]: Is this under debate?

**Comment by tjvr** [comment]
It might be better to add the new photo of Roger under a new filename, to make sure there are no caching issues.

**Comment by jamesdoc** [comment]
Netlify is pretty good. I doubt that will be an issue

File: pages/index.md (hunk 1 above)
Comment (jdfergusson) [reply pages/index.md]: Nope, just recommended not to be super obvious online apparently

**Pull request merged**